contracts/test/GnosisSafeABICreator.sol

```diff
@@ -3,5 +3,4 @@ pragma solidity ^0.8.15;
 
 // Getting ABIs for the Gnosis Safe master copy and proxy contracts
 import "@gnosis.pm/safe-contracts/contracts/GnosisSafe.sol";
-import "@gnosis.pm/safe-contracts/contracts/proxies/GnosisSafeProxy.sol";
```

Comment:
This contract is not used even on the ABI level after we switched to interface usage of Gnosis Safe